src/i965_drv_video.c

```diff
@@ -1078,6 +1078,27 @@ i965_GetConfigAttributes(VADriverContextP ctx,
 
             break;
 
+        case VAConfigAttribEncMaxSlices:
+            if (entrypoint == VAEntrypointEncSlice ||
+                entrypoint == VAEntrypointEncSliceLP) {
```

Comment:
@QuPengfei , The EncSliceLP is for VDENC. And the multi-slices are supported  although it is not exported.